Dockerfile

```diff
@@ -7,9 +7,14 @@ RUN go build ./cmd/eib
 
 
 # ----- Deliverable Image -----
-FROM registry.suse.com/bci/bci-base:15.5
+FROM opensuse/leap:15.5
```

Comment:
We will have to determine if we are comfortable shipping this as part of Edge 3.0. Personally, I think we are. It's not like this is a running service that depends on uptime. It being a standalone tool gives us more flexibility in using something like Leap, as well as how quickly we can release patches.